### PR TITLE
ci: bump to Go-1.16

### PR DESCRIPTION
**Pull request by sbinet**
Signed-off-by: Sebastien Binet <binet@cern.ch>

Please take a look.

### CODE REVIEW

File: .travis.yml [hunk 1]
```diff
@@ -5,8 +5,8 @@ go_import_path: gonum.org/v1/plot
 
 # Versions of go that are explicitly supported by gonum.
 go:
+ - 1.16.x
```
Comment (kortschak) [inline .travis.yml:8]: We can't have nice things on Windows.

Comment (sbinet) [reply .travis.yml:8]: indeed...

Comment (kortschak) [reply .travis.yml:8]: Not clear to me why I put this comment here, but glad you figured out what I meant.

**Review by kortschak — APPROVED** [review]

**Comment by kortschak** [comment]
Better now.

**Review by kortschak — APPROVED** [review]

**Pull request merged**